java-agent/instrumentation/rxjava-1/src/main/java/rx/__OpenTelemetryTracingUtil.java

```diff
@@ -3,7 +3,7 @@
 /**
  * This class must be in the rx package in order to access the package accessible onSubscribe field.
  */
-public class DDTracingUtil {
+public class __OpenTelemetryTracingUtil {
```

Comment:
Why prefixed?

Reply:
I was being excessively(?) worried about conflicting with something rxjava (or someone else) might add into the `rx.` package in the future

Reply:
It sort of matches the convention of prefixing field names with `__opentelemetry` when we inject fields into other peoples classes.

The name is also weird tho, I will change it to `OpenTelemetryTracingUtil` if you think that's better.

Reply:
TBH, it doesn't really matter to me.  I can live with it either way.